data/templates/centos-ks

```diff
@@ -45,14 +45,14 @@ clearpart --all --drives=<%=installDisk%>
         <% if(partition.mountPoint !== undefined && partition.size !== undefined) { %>
             <% if(partition.fsType !== undefined) { %>
                 <% if(partition.size === 'auto') { %>
-                    partition <%=partition.mountPoint%> --grow --fstype=<%=partition.fsType%>
+                    partition <%=partition.mountPoint%> --size=250 --grow --fstype=<%=partition.fsType%>
```

Comment:
@geoff-reid : could you please teach me why you set the minimum size to 250? What's the problem you are trying to resolve?